pkg/skaffold/initializer/init.go

```diff
@@ -35,10 +35,11 @@ import (
 
 // DoInit executes the `skaffold init` flow.
 func DoInit(ctx context.Context, out io.Writer, c config.Config) error {
+	var err error
```

Comment:
Remove local scope for this variable. This is not used to return in the end.

Reply:
this variable can get reused for each call that returns an `error` here so that we don't have to allocate a new one each time - I just forgot to change the `:=` to `=` in some places :) updated